specification/keyvault/resource-manager/Microsoft.KeyVault/stable/2022-07-01/common.json

```diff
@@ -0,0 +1,94 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "2022-07-01",
+    "title": "KeyVaultManagementClient",
+    "description": "The Azure management API provides a RESTful set of web services that interact with Azure Key Vault."
+  },
+  "paths": {},
+  "definitions": {
+    "CloudError": {
+      "description": "An error response from Key Vault resource provider",
+      "properties": {
+        "error": {
+          "$ref": "#/definitions/CloudErrorBody"
+        }
+      },
+      "x-ms-external": true,
+      "type": "object"
+    },
+    "CloudErrorBody": {
+      "description": "An error response from Key Vault resource provider",
+      "properties": {
+        "code": {
+          "type": "string",
+          "description": "Error code. This is a mnemonic that can be consumed programmatically."
+        },
+        "message": {
+          "type": "string",
+          "description": "User friendly error message. The message is typically localized and may vary with service version."
+        }
+      },
+      "x-ms-external": true,
+      "type": "object"
```

Comment:
I am curious , why did you need to add the type:object filed everywhere ?

Reply:
They were added for parity with the previous stable ApiVersion (2021-10-01), which contains these type: object fields everywhere.